using System;

// Использование нескольких catch-инструкций
// System.Exception

namespace ExceptionSix
{
   internal class Program
   {
      static void Main()
      {
         Console.WriteLine("Hello World!");
      }
   }
}
